regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -112,6 +113,7 @@ var CommentReviewView = Backbone.View.extend({
         }).get();
         toggleInput($text, $valid.length === 0);
         toggleInput($select, $valid.length > 0);
+        toggleInput($selectDropdown, $valid.length > 0);
```

Comment:
`toggleInput` sets the "disabled" attribute, which isn't what you want here. Let's step back and think about a better strategy. @jmcarp what are your thoughts?